guides/source/action_controller_overview.md

```diff
@@ -15,6 +15,7 @@ After reading this guide, you will know:
 * How to stream data directly to the user's browser.
 * How to filter sensitive parameters, so they do not appear in the application's log.
 * How to deal with exceptions that may be raised during request processing.
+* About the built-in health check end-point useful for load balancers and uptime monitors.
```

Comment:
```suggestion
* How to use the built-in health check end-point with load balancers and uptime monitors.
```

Should we keep all of these `How to ...`?

Reply:
I intentionally wanted to break up to how-to-chain :joy: basically if we are going to repeat it, then we can prefix that whole list with something that makes sense.. like:

In this guide you will learn how to:

* Stream data directly to the user's browser.
* ...